README.md

```diff
@@ -18,6 +18,22 @@ The default port for the service is 9090.
 ### Usage
 ```pip install . && swift-x-account-sharing```
 
+The swift-x-account-sharing can use two different database styles for
+storing the sharing information. The software defaults to an in-memory
+database implemented in Python, but the preferred alternative for
+production is PostgreSQL.
+
+The PostgreSQL implementation can be enabled via environment variables,
+that are
+* SHARING_DB_POSTGRES; if found in environmen, uses PostgreSQL.
+* SHARING_DB_PASSWORD; REQUIRED; The password used in connecting to the
```

Comment:
```suggestion
* SHARING_DB_PASSWORD; REQUIRED; The password used for connecting to the
```
probably it sounds better `used for` than `used in`, also consistent with the env var description below